### ISSUE TITLE
CoC: Adding a list of violating actions

### ISSUE THREAD

**Issue by choldgraf** [jupyter/governance#70]
In working on #69 , I found that the [Turing Way CoC](https://github.com/alan-turing-institute/the-turing-way/blob/master/CODE_OF_CONDUCT.md#4-enforcement-manual) had a useful list of unnacceptable behaviors and some consequences:

https://github.com/alan-turing-institute/the-turing-way/blob/master/CODE_OF_CONDUCT.md#22-unacceptable-behaviour

I found this to be a way of making the more abstract ideals of the CoC a little bit more concrete. I also found their list to be generally useful and complete.

What do people think about me copying that section, with minimal changes, from the Turing Way CoC into the Jupyter CoC?

For reference, here is the full text of the section:

---

### 2.2 Unacceptable Behaviour

Examples of unacceptable behaviour by Turing Way community members at any project event or platform include:

* written or verbal comments which have the effect of excluding people on the basis of membership of any specific group
* causing someone to fear for their safety, such as through stalking, following, or intimidation
* violent threats or language directed against another person
* the display of sexual or violent images
* unwelcome sexual attention
* nonconsensual or unwelcome physical contact
* sustained disruption of talks, events or communications
* insults or put downs
* sexist, racist, homophobic, transphobic, ableist, or exclusionary jokes
* excessive swearing
* incitement to violence, suicide, or self-harm
* continuing to initiate interaction (including photography or recording) with someone after being asked to stop
* publication of private communication without consent

### 2.3 Consequences of Unacceptable Behaviour

Participants who are asked to stop any inappropriate behaviour are expected to comply immediately.
This applies to all Turing Way community events and platforms, either online or in-person.
If a participant engages in behaviour that violates this Code of Conduct, any member of the core development team may warn the offender, ask them to leave the event or platform (without refund), or impose any other appropriate sanctions (see the [enforcement manual](#enforcement-manual) for details).

**Comment by damianavila** [comment]
> What do people think about me copying that section, with minimal changes, from the Turing Way CoC into the Jupyter CoC?

I am +1 on this but highlighting these are not the only cases. We want to cover other unacceptable behaviours that are not described in that section, IMHO.

**Comment by choldgraf** [comment]
@damianavila I totally agree with you that this isn't an exhaustive list. Perhaps rather than trying to define all of the different kinds of unacceptable behavior, we should just add in a sentence explicitly saying "this is an incomplete list, and specific examples are given as guidance for the kind of behavior we do not tolerate, as opposed to an exhaustive list."

**Comment by damianavila** [comment]
> "this is an incomplete list, and specific examples are given as guidance for the kind of behavior we do not tolerate, as opposed to an exhaustive list."

Sounds great to me.
